frontend/javascripts/oxalis/model/accessors/dataset_accessor.js

```diff
@@ -83,19 +87,32 @@ export class ResolutionInfo {
   getResolutionByIndexOrThrow(index: number): Vector3 {
     const resolution = this.getResolutionByIndex(index);
     if (!resolution) {
-      throw new Error(`Resolution with in index ${index} does not exist`);
+      throw new Error(`Resolution with index ${index} does not exist.`);
     }
     return resolution;
   }
 
-  getResolutionByIndexWithFallback(index: number): Vector3 {
-    const resolutionMaybe = this.getResolutionByIndex(index);
+  getResolutionByIndexWithFallback(
+    index: number,
+    fallbackResolutionInfo: ?ResolutionInfo,
+  ): Vector3 {
+    let resolutionMaybe = this.getResolutionByIndex(index);
     if (resolutionMaybe) {
       return resolutionMaybe;
-    } else {
-      const powerOf2 = this.indexToPowerOf2(index);
-      return [powerOf2, powerOf2, powerOf2];
     }
+
+    resolutionMaybe =
+      fallbackResolutionInfo != null ? fallbackResolutionInfo.getResolutionByIndex(index) : null;
+    if (resolutionMaybe) {
+      return resolutionMaybe;
+    }
+
+    if (index === 0) {
+      // If the index is 0, only mag 1-1-1 can be meant.
+      return [1, 1, 1];
+    }
+
+    throw new Error("");
```

Comment:
Reminder that this issue exists. Could it be an option to return the isotropic resolution here as you did before for now?

Reply:
I checked the callers of this method and this should not be called for not-existing magnifications (except for mag 1 which is kind of a special case, since our global positioning system is always in mag 1). So, the error should not be thrown in theory. Since returning the isotropic resolution is potentially wrong, I'd prefer to simply throw the error and fix the call site once we see this error in the wild. Do you agree?

Reply:
Yes agreed, but I would add an error message, then :)

Reply:
Done ^^